website_portal/controllers/main.py

```diff
@@ -93,3 +94,16 @@ def details_form_validate(self, data):
             error_message.append(_('Some required fields are empty.'))
 
         return error, error_message
+
+
+class Website(openerp.addons.web.controllers.main.Home):
+
+    @http.route(website=True, auth="public")
+    def web_login(self, redirect=None, *args, **kw):
+        res = super(Website, self).web_login(redirect=redirect, *args, **kw)
+        if request.session.uid:
+            user = request.env['res.users'].browse(request.session.uid)
```

Comment:
@pedrobaeza Can you help me??
Why is not the same result:??
request.env.user.browse(13).has_group('base.group_user') = {bool} True
request.env['res.users'].browse(13).has_group('base.group_user') = {bool} False

This is the reason... :flushed: